draft-ietf-scitt-architecture.md

```diff
@@ -672,7 +672,9 @@ Relying Parties can choose which Issuers they trust.
 
 Multiple Issuers can make the same Statement about a single Artifact, affirming multiple Issuers agree.
 
-At least one identifier representing one credential MUST be included in the protected header of the COSE Envelope, as one of `x5t`, `x5chain` or `kid`.
+At least one identifier representing one credential MUST be included in the protected header of the COSE Envelope.
+
+For example, when x.509 is utilized, either `x5t`, `x5chain` or `kid` must be included in the protected header.
```

Comment:
```suggestion
```

Reply:
Why the deletion? Is this a deferral to the COSE spec?

Reply:
https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/336#discussion_r1941469555